source/llassetgen-rendering/WindowQt.h

```diff
@@ -52,6 +52,7 @@ class WindowQt : public QWindow {
     virtual void fontColorGChanged(QString value);
     virtual void fontColorBChanged(QString value);
     virtual void resetTransform3D();
+    virtual void superSamplingChanged(int index);
```

Comment:
Couldn't this method be defined on the subclass in `main.cpp`? I don't see it used anywhere in the base class.

Reply:
That's true, and I already tried that. Seems that Qt couldn't find thos functions as `slots` if they are not defined in the base class. Adding `Q_OBJECT` to the subclass in `main.cpp` results in an error I couldn't fix.
It all comes down to the problem, that some Qt classes want to use their own gl, but I import the gl functions using glbinding. So I have to split the functionality to thos 2 subclasses. Confused? #MeToo.